host/xtest/stats.c

```diff
@@ -154,6 +155,15 @@ static int stat_alloc(int argc, char *argv[])
 		       stats[n].desc);
 		printf("Bytes allocated:                       %"PRId32"\n",
 		       stats[n].allocated);
+		if (stats[n].free2_sum && stats[n].size != stats[n].allocated) {
+			double free2_sum = stats[n].free2_sum;
+			double free_sum = stats[n].size - stats[n].allocated;
+			double free_quote = sqrt(free2_sum) / free_sum;
+			double fragmentation = 1.0 - free_quote;
```

Comment:
From [metric_for_memory_fragmentation post](https://asawicki.info/news_1757_a_metric_for_memory_fragmentation), fragmentation metric should be `1.0 - free_quote * free_quote`.

Reply:
You're right, I'll update.